source/ui/dataman/AMScanView.cpp

```diff
@@ -689,48 +763,26 @@ AMScanViewExclusiveView::AMScanViewExclusiveView(AMScanView* masterView) : AMSca
 	gl->setSizePolicy(sizePolicy());
 	setLayout(gl);
 
-
 	// the list of plotItems_ needs one element for each scan.
+
 	for(int scanIndex=0; scanIndex < model()->scanCount(); scanIndex++) {
 		addScan(scanIndex);
 	}
 
+	// Make connections.
+
 	connect(model(), SIGNAL(exclusiveDataSourceChanged(QString)), this, SLOT(onExclusiveDataSourceChanged(QString)));
+	connect( plot_->plot()->signalSource(), SIGNAL(dataPositionChanged(QPointF)), this, SIGNAL(dataPositionChanged(QPointF)) );
 
 	reviewPlotAxesConfiguration(plot_);
 	refreshTitle();
 }
 
 AMScanViewExclusiveView::~AMScanViewExclusiveView() {
-	// PlotSeries's will be deleted as children items of the plot.
-
-	delete plotCursor_;
 	plot_->deleteLater();
```

Comment:
Don't we need to delete `dragZoomerTool_, wheelZoomerTool_  and dataPositionTool_` here?
`